### PR TITLE
Update SSSOM mapping files

### PR DESCRIPTION
**Pull request by rays22**
Following [SOP here.](https://obophenotype.github.io/bio-attribute-ontology/editors-guide/)

### CODE REVIEW

**Comment by matentzn** [comment]
Why is the diff so large?

**Comment by rays22** [comment]
> Why is the diff so large?

I have updated some term labels and fixed an incorrect mapping.

**Comment by rays22** [comment]
> Why is the diff so large?

It looks like that somebody has also updated the match_type on every line of the vt sssom table.

File: src/mappings/oba-efo-mapping-exclusions.sssom.tsv [hunk 1]
```diff
@@ -1,2 +1,6 @@
 subject_id	subject_label	predicate_id	object_id	match_type	comment
-EFO:0005514	HER2 status	skos:exactMatch	sssom:NoMapping	HumanCurated	tumor level measurements out of scope for OBA
\ No newline at end of file
+EFO:0005514	HER2 status	skos:exactMatch	sssom:NoMapping	HumanCurated	tumor level measurements out of scope for OBA
```
Comment (matentzn) [inline src/mappings/oba-efo-mapping-exclusions.sssom.tsv:2]: It is important that this is excellent suggestion is communicated in your meetings with GWAS.

**Review by matentzn — CHANGES_REQUESTED** [review]
Please reduce the diff, the mapping_justification should be always `semapv:ManualMappingCuration` and I am worried that I made some changes to these files without updating them on Google Docs (during the paper writing)

**Comment by rays22** [comment]
> Please reduce the diff, the mapping_justification should be always `semapv:ManualMappingCuration` and I am worried that I made some changes to these files without updating them on Google Docs (during the paper writing)

I have reduced the size of the diff by using `semapv:ManualMappingCuration` for mapping_justification.

**Review by matentzn — APPROVED** [review]
Looks great!

**Pull request merged**